app/views/admin/users/show.html.slim

```diff
@@ -0,0 +1,41 @@
+section.section
+  .columns
+    .column
+      h1.title #{@user.first_name || @user.last_name ? @user.name + " - " : ""} #{@user.email}
```

Comment:
How about this instead? I think it's the same result..

```
#{[@user.name, @user.email].join(" - ")}
```

Reply:
Will fix.